source/Octopus.Manager.Tentacle.Tests/Octopus.Manager.Tentacle.Tests.csproj

```diff
@@ -0,0 +1,112 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Import Project="..\packages\NUnit3TestAdapter.4.5.0\build\net462\NUnit3TestAdapter.props" Condition="Exists('..\packages\NUnit3TestAdapter.4.5.0\build\net462\NUnit3TestAdapter.props')" />
```

Comment:
What is all this about? I didn't see it in the other test projects

Reply:
Those seem to be generated by Rider, let me look into it